testing/src/test/java/io/stargate/it/backends/Cassandra311Backend.java

```diff
@@ -24,6 +24,6 @@
 public class Cassandra311Backend extends AllBackendSuite {
   @Parameterized.Parameters(name = "{index}: {0}")
   public static Iterable<Object[]> functions() {
-    return Collections.singletonList(new Object[] {"cassandra:3.11.6", false, "3.11"});
+    return Collections.singletonList(new Object[] {"cassandra:3.11.6", false, "3.11", 3});
```

Comment:
With this approach, all our tests will run with 3 stargate nodes, it is not ideal but other solutions turn out to be not feasible/optimal: 
i.e.: creating `Cassandra311BackendMulitpleNodes` and `BackandSuiteMultipleNodes` with only one test: `MultipleStargateInstancesTest` will lead to code duplication and two suites to extend by the other persistence implementations.

Reply:
If there are limitations in passing parameters to the suite, then maybe there could be a `Starter addStargateNode()` method that the inherited test class could use to create additional stargate nodes?

Reply:
This would be useful for CQL tests that I need to write to test `system.peers` functionality.

Reply:
right now the suite with 3 nodes seems working (there is sometimes the race condition (bind address already in use) on Jenkins when starting the last node). The `addStargate()` called by the actual test method is problematic because of the stateful logic and changing the IPs if:
```
  if (Boolean.getBoolean(SKIP_HOST_NETWORKING_FLAG)) {
```
I am not sure if this MAC code is working, I was not able to make the Mac part work for me, @jorgebay neither.
The 2nd issue with this approach is static state and the fact that test are not isolated. So starting new starter host will impact other ITs. Ideally, we would have `Junit rule` or something similar for `Stargate` that will be spin per test class suite and isolated from eachother.